pkgs/development/python-modules/contextvars/default.nix

```diff
@@ -0,0 +1,21 @@
+{ lib, buildPythonPackage, fetchPypi, isPy36, immutables }:
+
+buildPythonPackage rec {
+  pname = "contextvars";
+  version = "2.2";
+  disabled = !isPy36;
+
+  src = fetchPypi {
+    inherit pname version;
+    sha256 = "046b385nfzkjh0wqmd268p2jkgn9fg6hz40npq7j1w3c8aqzhwvx";
+  };
+
+  propagatedBuildInputs = [ immutables ];
+
+  meta = {
+    description = "This package implements a backport of Python 3.7 contextvars module for Python 3.6.";
```

Comment:
No period: https://nixos.org/nixpkgs/manual/#sec-standard-meta-attributes
And for the purpose of brevity, I'd leave out "This package implements"/